src/main/java/lesson8Person/Person.java

```diff
@@ -0,0 +1,71 @@
+package lesson8Person;
+
+import java.time.*;
+import java.util.Objects;
+
+public class Person {
+    private final String name;
+    private final String surname;
+
+    // if I don't initialize these, I can't use the 'LocalDate.of' method, any solutions to that?
+    private int yearOfBirth = 0;
+    private Month monthOfBirth = Month.January;
```

Comment:
Niepotrzebnie inicjalizujesz to pole, ponieważ twój jedyny konstruktor tak czy inaczej je ustawia.

Reply:
gdy nie inicjuje tych zmiennych, program się nie kompiluje, ponieważ java zakłada, że te pola są null i nie jest w stanie wykonać linijki w której inicjuję this.birthDay. Naniosłem zmianę, usunąłem te składowe z obiektu, pozostając przy birthDay.